src/client/index.ts

```diff
@@ -109,7 +109,6 @@ import {
 import { BaseRequest, BaseResponse } from '../models/methods/baseMethod'
 import combine from '../transaction/combine'
 import { sign } from '../transaction/sign'
-import { deriveAddress, deriveXAddress } from '../utils/derive'
```

Comment:
If `deriveAddress` is being removed, can we remove the code from `'../utils/derive'`?